src/main/scala/akka/contrib/process/BlockingProcess.scala

```diff
@@ -107,16 +110,24 @@ class BlockingProcess(
     }
 
     try {
-      val stdin =
-        context.actorOf(OutputStreamSubscriber.props(process.getOutputStream), "stdin")
-
-      val stdout =
-        context.watch(context.actorOf(InputStreamPublisher.props(process.getInputStream, stdioTimeout), "stdout"))
-
-      val stderr =
-        context.watch(context.actorOf(InputStreamPublisher.props(process.getErrorStream, stdioTimeout), "stderr"))
-
-      context.parent ! Started(ActorSubscriber(stdin), ActorPublisher(stdout), ActorPublisher(stderr))
+      val stdin = Sink.outputStream(process.getOutputStream)
+        .mapMaterializedValue { futureResult =>
+          futureResult.onComplete(_ => context.stop(self))
```

Comment:
@2m - I've also confirmed merging of `Source` does work like what you said:

https://github.com/2m/akka-contrib-extra/commit/35faea520dca7d21abcc03b1c3598c7416039547#diff-ac1a75a9b3c83841178d1c3d7d22c210R124

Turns out I need to shutdown the `BlockingProcess` actor at the end of the Sink for stdin, Source for stdout, or Source for stderr.

Somehow I don't need to do this before when `InputStreamPublisher` and `OutputStreamPublisher` was used. I've looked around, but I can't seem to find where the `BlockingProcess` would be terminated.